docs/src/main/sphinx/connector/join-pushdown-enabled-true.fragment

```diff
@@ -0,0 +1,33 @@
+Cost-based join pushdown
+""""""""""""""""""""""""
+
+The connector supports cost-based :ref:`join-pushdown` to make intelligent
+decisions about whether to push down a join operation to the data source.
+
+When cost-based join pushdown is enabled, the connector only pushes down join
+operations if the available :doc:`/optimizer/statistics` suggest that doing so
+improves performance. Note that if no table statistics are available, join
```

Comment:
Is there someplace we can link to which explains table stats and where/how to collect those?

Reply:
The docs for each connector that includes table stats .. where  this fragment is used .. includes docs for table stats just about pushdown in the same performance section..

Reply:
+1 to Manfred, by intention with this restructure the table statistics should always immediately precede this pushdown information on a connector doc. We also have the link to `/optimizer/statistics` to give information about table stats in general.